onnx/defs/math/defs.cc

```diff
@@ -3660,4 +3660,534 @@ ONNX_OPERATOR_SET_SCHEMA(
             propagateShapeFromInputToOutput(ctx, 0, 1);
           }
         }));
+
+
+template <typename T>
+static T get_scalar_value_from_tensor(const ONNX_NAMESPACE::TensorProto* t) {
+  if (t == nullptr) {
+    return T{};
+  }
+
+  auto data_type = t->data_type();
+  switch (data_type) {
+    case ONNX_NAMESPACE::TensorProto::FLOAT:
+      return static_cast<T>(ONNX_NAMESPACE::ParseData<float>(t).at(0));
+    case ONNX_NAMESPACE::TensorProto::DOUBLE:
+      return static_cast<T>(ONNX_NAMESPACE::ParseData<double>(t).at(0));
+    case ONNX_NAMESPACE::TensorProto::INT32:
+      return static_cast<T>(ONNX_NAMESPACE::ParseData<int32_t>(t).at(0));
+    case ONNX_NAMESPACE::TensorProto::INT64:
+      return static_cast<T>(ONNX_NAMESPACE::ParseData<int64_t>(t).at(0));
+    default:
+      fail_shape_inference("Unsupported input data type of ", data_type);
+  }
+}
+
+static const char* DFT_ver13_doc =
+    R"DOC(Computes the discrete Fourier transform of input.)DOC";
+
+    ONNX_OPERATOR_SET_SCHEMA(DFT,
+        13,
```

Comment:
The latest opset version is 16 so please replace all 13 with 16.

You need to run `tools/update_doc.sh` or `tools/update_doc.bat` to get updated documents (Changelog.md, Operators.md, TestCoverage.md) and please include them in this PR. Thanks!